react/features/invite/functions.ts

```diff
@@ -571,98 +571,86 @@ export function searchDirectory( // eslint-disable-line max-params
  * @param {IReduxState} state - The current state.
  * @param {string} inviteUrl - The conference/location URL.
  * @param {boolean} useHtml - Whether to return html text.
+ * @param {boolean} skipDialIn - Whether to skip dial-in options or not.
  * @returns {Promise<string>} A {@code Promise} resolving with a
  * descriptive text that can be used to invite participants to a meeting.
  */
-export function getShareInfoText(state: IReduxState, inviteUrl: string, useHtml?: boolean): Promise<string> {
+export function getShareInfoText(
+        state: IReduxState, inviteUrl: string, useHtml?: boolean, skipDialIn?: boolean): Promise<string> {
     let roomUrl = _decodeRoomURI(inviteUrl);
-    const includeDialInfo = state['features/base/config'] !== undefined;
 
     if (useHtml) {
         roomUrl = `<a href="${roomUrl}">${roomUrl}</a>`;
     }
 
     let infoText = i18next.t('share.mainText', { roomUrl });
 
-    if (includeDialInfo) {
-        const { room } = parseURIString(inviteUrl);
-        let numbersPromise;
-        let hasPaymentError = false;
-
-        if (state['features/invite'].numbers
```

Comment:
I removed this check because it contradicts the comment below. Since we get a URL we need to check the values for that one specifically.